### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by Molkree** [orcasound/orca-action-workflow#32]
- Start with repo readme, is it already outdated.
- Maybe create directories for each data source/algorithm, each with a short how-to readme?

**Comment by valentina-s** [comment]
Yes, main readme can describe the purpose (since not everybody understands github actions), and then you can have links to subfolder readmes with more details. It would be good to have instructions like:
* here is how to run the workflow with default parameters
* here is where to change stuff to run the workflow with new parameters.

**Comment by sahilempire** [comment]
I want to work on this issue, I have checked all the things which should be present in the README file. 
So please assign me, to this issue. @Molkree
